paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -1622,6 +1685,75 @@ class PaymentSheet internal constructor(
          */
         fun confirm()
 
+        /**
+         * Builder utility to set optional callbacks for [PaymentSheet.FlowController].
+         *
+         * @param resultCallback Called when a [PaymentSheetResult] is available.
+         * @param paymentOptionCallback Called when the customer's desired payment method changes.
+         */
+        class Builder(
+            internal val resultCallback: PaymentSheetResultCallback,
+            internal val paymentOptionCallback: PaymentOptionCallback
+        ) {
+            internal var externalPaymentMethodConfirmHandler: ExternalPaymentMethodConfirmHandler? = null
```

Comment:
Do these need to be internal rather than private?

Reply:
They need to be internal so they are visibile to `rememberPaymentSheetFlowController`